e2e/cypress/integration/account_settings/general/fullname_spec.js

```diff
@@ -117,3 +117,78 @@ describe('Account Settings -> General -> Full Name', () => {
         cy.get('#nameDesc').should('be.visible').should('contain', testUser.first_name + '_new ' + testUser.last_name);
     });
 });
+
+describe('MM-30354 Automate backlogs - Account Settings > Full Name', () => {
```

Comment:
Btw, can we move this entire block into its own separate file so that only these tests will undergo observation period whether tests are stable to be considered for production? If going to stay here, then `// Stage: @prod` on top of this file needs to be removed.

Reply:
sure, I'll move to a different file, i just made up the new name but feel free to suggest a more accurate one if you feel it might cause confussion